src/extension/features/general/hide-help/index.js

```diff
@@ -18,11 +18,10 @@ export class HideHelp extends Feature {
     let hide = getToolkitStorageKey('hide-help');
 
     if (hide === null) {
-      setToolkitStorageKey('hide-help', 'true');
-      hide = 'true';
+      setToolkitStorageKey('hide-help', true);
```

Comment:
Does `hide` still need to be set to true here for the upcoming compare on line 24?

Reply:
actually I should change line 18 to be const hide = getToolkitStorageKey('hide-help', true) which will default it to true if it's not stored

Reply:
Yeah, that'd work too.